### DISCUSSION TITLE
Plex and first artist problem

### DISCUSSION THREAD

**Discussion by StefanDorschu** [beetbox/beets#4853] [General]
I'm using plex to host my library, the problem is (or i don't know how to avoid that) that plex does not use the first artist of an album (instead showing multiple artists like artist, artist feat. xyz)

During import i am able to generate a folder structure that uses the first artist (so removing feat. e.g.). Is there a way to modify the album_artist tag in general using the beets_config_file?

Thank you so much,

Stefan

**Comment by sampsyo** [comment]
You might be able to accomplish this with a funky combination of `beet modify` and something like the `inline` plugin. On master, you can do a "formatted modify" as described here:
https://github.com/beetbox/beets/blob/master/docs/reference/cli.rst#modify

So something resembling `beet modify albumartist='$first_albumartist'` would work, if only we had a way to define `$first_albumartist`. You could pair this with an `inline` plugin that does some string manipulation on `$albumartist` to try to heuristically split out the first artist.

**Reply by StefanDorschu** [reply]
Thank you for that info!
